src/sentry/rules/conditions/event_frequency.py

```diff
@@ -75,11 +90,22 @@ def get_rate(self, event, interval, environment_id):
         end = timezone.now()
         return self.query(event, end - duration, end, environment_id=environment_id)
 
+    @property
+    def is_guessed_to_be_created_on_project_creation(self):
+        """
+        Best effort approximation on whether a rule with this condition was created on project creation based on how
+        closely the rule and project are created; and if the label matches the default name used on project creation.
+        :return:
+            bool: True if rule is approximated to be created on project creation, False otherwise.
+        """
+        delta = abs(self.project.date_added - self.rule.date_added)
```

Comment:
The rule will always be created after the project, so we probably want to subtract the project creation from the rule creation instead. Otherwise this delta will always be negative, and so this condition will always pass.

Reply:
Posting for easy reference next time, abs works on timedelta objects.
https://docs.python.org/2/library/datetime.html#timedelta-objects

Reply:
Resolved offline